content/influxdb/cloud/visualize-data/visualization-types/map.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Map visualization
+list_title: Map
+list_image: /img/influxdb/2-0-visualizations-map-point-example.png
+description: >
+  The Map visualization displays geo-temporal data on a geographic map.
+weight: 206
+menu:
+  influxdb_cloud:
+    name: Map
+    parent: Visualization types
+---
+
+The **Map** visualization displays geo-temporal data on a geographic map.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-map-point-example.png" alt="Map data visualization" />}}
+
+## Set up the Map visualization
+
+To view geo-temporal data on a geographic map, set up a Map visualization.
+
+### Set up the Map visualization
+
+1.  Do one of the following:
+
+    - Click  **Data Explorer** in the navigation bar.
+
+        {{< nav-icon "data-explorer" >}}
+
+    - Click **Dashboards** in the navigation bar:
+
+        {{< nav-icon "dashboards" >}}
+
+        Click the name of the dashboard to update and then do one of the following:
+
+        - **To edit an existing cell**, click the **{{< icon "cog" >}}** icon on
+          the cell and then **{{< icon "edit" >}} Configure**.
+        - **To create a new cell**, click **{{< icon "add-cell" >}} Add Cell**.
+
+2.  Use the **Query Builder** or the **Script Editor** to enter your query.
+    To determine the location of points on a geographic map, query results must
+    include the following **fields**:
+
+    - **lat**: latitude in decimal degrees (WGS 84)
+    - **lon**: longitude in decimal degrees (WGS 84)
+
+    _**Or**_ the  `s2_cell_id` **tag**  ([S2 Cell ID](https://s2geometry.io/devguide/s2cell_hierarchy.html#s2cellid-numbering) as a token)
+
+    {{% note %}}
+If query results include `lat` and `lon` fields _and_ an `s2_cell_id` tag,
+the map uses the `lat` and `lon` fields to determine point locations.
+If results only include the `s2_cell_id`, the map uses the center of the S2 cell
+as the point location.
+    {{% /note %}}
+
+    _See [Example queries](#example-queries)._
+
+<!-- 3.  Select the **Map** option from the visualization drop-down list in the upper left,
```

Comment:
@Palakp41 @sanderson, shall we uncomment this step (or any part of it) before publishing? Looks like we've added the Map option to production.